mu4/playback/internal/playbackcontroller.cpp

```diff
@@ -33,9 +38,28 @@ void PlaybackController::init()
         onNotationChanged();
     });
 
-    audioPlayer()->statusChanged().onReceive(this, [this](const PlayStatus&) {
+    sequencer()->statusChanged().onReceive(this, [this](const audio::ISequencer::Status&) {
         m_isPlayingChanged.notify();
     });
+    sequencer()->initMIDITrack(MIDI_TRACK);
+
+    switch (m_cursorType) {
+    case SMOOTH:
+        sequencer()->positionChanged().onNotify(this, [this]() {
```

Comment:
This position does not take into account the latency in the fifo buffer, i.e. it is ahead of what is actually being sound.
If we need to increase the buffer size, then this will become a serious problem.